docs/content/basics/screenshot_testing.md

```diff
@@ -1,5 +1,86 @@
+If the generated snapshot file from the test differs from the reference, the test fails, passes otherwise.
+
+## Motivation
+Screenshot tests are easy to write and maintain and run fast (≈ 1 sec per test), what makes them affordable to execute upon PRs.
+They focus on detecting visual bugs that standard UiAutomator and Espresso cannot, namely
+
+1. Ui defects introduced by updates in libraries like Material design and Constraint Layout
+   
+2. Spacing, styles and themes (light mode vs. dark mode)
+   
+3. Layout correctness:
+     - Multiline text does not cut off
+     - No view overlapping on visibility changes
+     - Proper view alignment under different screen densities and/or font sizes
+     - Text, icons and Ui alignment on RTL & LTR languages, among other language related issues
+
+## How it works
+First of all, we need to understand that snapshot testing process differs from standard testing.
+Most snapshot testing frameworks provide two main tasks:
+
+1. **Record**: Generates a snapshot file. Executed locally. this file will be reviewed by our peers, and once approved, uploaded to the CI as our source of truth. All further verification fo that test will be made against it.
+2. **Verify**: When executed, it generates a snapshot file for each test that will be compared, pixel by pixel, with the one taken as reference when recording. Executed on the CI.
+
+Once everybody in the team has agreed the same device setup for snapshot testing, the process goes as follows:
+
+1. Create a test that snapshots a view. This could range from a view dynamically inflated to the activity that is shown on the emulator/device screen.
+2. Make sure adb only lists the snapshot testing device/emulator. That is because instrumentation tests run on all emulators/devices adb detects. This might cause some issues like snapshot file duplication errors, longer running times, and the like.
+3. Execute "record" locally to generate the corresponding snapshot files and open the corresponding PR containing them.
+4. Some teammates review the snapshots to confirm that they reflect the state of the view we are expecting. Once approved and merged, this Snapshot is taken as source of truth for future verifications of the tests.
+5. Verify the snapshots periodically (e.g. on every PR, once a day, etc.). Ideally you have a CI with a job configured to "verify" them. This means, it executes the snapshot tests with the most up-to-date code, and takes a screenshot. After that, it compares this screenshot with the last one uploaded to the CI or the one being pushed in the PR. If they both match, the test passes, fails otherwise.
+
+
+## Challenges
+Unfortunately, Screenshot tests also come with their own problems. 
+
+1. Flakiness: Same as with Ui tests, Snapshot test face flakiness problems. On one hand, we face similar problems to Ui Testing. Most of the issues described in the [flakiness](../practices/flakiness.md) section also apply to Ui tests.
```

Comment:
`section also apply to Ui tests`

->

`section also apply to Screenshot tests`?